src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -34,13 +34,18 @@ const logToSentry = data => {
   return isCaptured;
 };
 
-const sanitizeAuthn = authnCtx => authnCtx.replace(MHV_SKIP_DUPE, '');
+export const sanitizeAuthn = authnCtx => authnCtx.replace(MHV_SKIP_DUPE, '');
```

Comment:
can we go ahead and add a test case around this. I think we just add it to the existing test file until we decide on how we want to break our tests up

Reply:
I can absolutely add some tests cases for the sanitizeAuthn function.  I expect it will change and or be removed when we grab the different headers from IAM.